frigate/output.py

```diff
@@ -310,11 +315,14 @@ def update_frame(self):
 
         return True
 
-    def update(self, camera, object_count, motion_count, frame_time, frame) -> bool:
+    def update(self, camera, camera_config, object_count, motion_count, frame_time, frame) -> bool:
+        # don't process if birdseye is disabled for this camera
+        if camera_config and not camera_config.enabled:
```

Comment:
The config parser should make it impossible for `camera_config` to be None here.

```python
if not camera_config.enabled:
```